src/OrleansServiceBus/Providers/Streams/EventHub/EventHubPartitionCheckpoint.cs

```diff
@@ -0,0 +1,74 @@
+﻿
+using System;
+using System.Threading.Tasks;
+using Microsoft.ServiceBus.Messaging;
+using Orleans.AzureUtils;
+
+namespace Orleans.ServiceBus.Providers
+{
+    internal class EventHubPartitionCheckpoint
```

Comment:
BTW, maybe even put these classes in a sub-folder and sub namespace that denotes a dependency on azure storage